tests/acceptance/features/coreApiAuth/webDavSpecialURLs.feature

```diff
@@ -20,7 +20,7 @@ Feature: make webdav request with special urls
       | /remote.php//dav/files/%username%/PARENT/parent.txt |
       | /remote.php//webdav/PARENT                          |
       | //remote.php/dav//files/%username%//FOLDER          |
-    Then the HTTP status code of responses on each endpoint should be "200,200,204,204,200" respectively
+    Then the HTTP status code of responses on each endpoint should be "200,200,204,204,200" on oCIS and "204,204,204,204,204" on reva
```

Comment:
should we use `or` here because only one of the two will be true at a given test context
```suggestion
    Then the HTTP status code of responses on each endpoint should be "200,200,204,204,200" on oCIS or "204,204,204,204,204" on reva
```

Reply:
good suggestion - done